src/components/Header/Header.css

```diff
@@ -0,0 +1,38 @@
+.header-container {
+  margin-top: 43px;
```

Comment:
- [ ] Remove the margin-top and apply a height for the entire header.

Reply:
The changes were made in the scss but it dont compile the css . Fixed the issue